samples/bluetooth/broadcaster_multiple/src/broadcaster_multiple.c

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/sys/printk.h>
+#include <zephyr/bluetooth/bluetooth.h>
+
+/* Maximum supported AD data length, use a value supported by the Controller,
+ * Bluetooth Core Specification define minimum of 31 bytes will be supported by
+ * all Controllers, can be a maximum of 1650 bytes when supported.
+ */
+#if defined(CONFIG_BT_CTLR_ADV_DATA_LEN_MAX)
+#define BT_AD_DATA_LEN_MAX CONFIG_BT_CTLR_ADV_DATA_LEN_MAX
+#else
+#define BT_AD_DATA_LEN_MAX 31U
+#endif
```

Comment:
Since `CONFIG_BT_CTLR_ADV_DATA_LEN_MAX` is only guarded by `CONFIG_BT_CTLR_ADV_EXT`, and since this sample requires extended advertising, I think this guard here can be removed, right?

Reply:
Ah, I see. 

I guess the host should actually read the maximum supported advertising length with `BT_HCI_OP_LE_READ_MAX_ADV_DATA_LEN` in those cases, but it isn't implemented in the Zephyr host. 

It seems that I even created an issue for this a while ago: https://github.com/zephyrproject-rtos/zephyr/issues/45230